aut-apps.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+echo 'Instal Googe Chrome'
+sudo bash -c "touch /etc/apt/sources.list.d/google-chrome.list && echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main'>/$
+wget https://dl.google.com/linux/linux_signing_key.pub
+sudo bash -c "apt-key add linux_signing_key.pub && apt update && apt install google-chrome-stable"
```

Comment:
Don't use `sudo` inside of this script but just require from the user to execute the script itself as root